packages/contracts/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './artifacts';
+import * as wrappers from './wrappers';
+export { wrappers };
```

Comment:
Also, in `contracts` let's add a `.npmignore` file and make sure the bundle published to `npmjs.com` repository only includes these files and not the `tests` and `Solidity` source dirs.

Reply:
👍 I was planning to sort what to include in the published package out at the end. Want to make sure everything works inside the monorepo first.